### PR TITLE
feat: add support for multi attribute in partial entity attributes update #101

### PR DESCRIPTION
**Pull request by HoucemKacem**

### CODE REVIEW

File: entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt [hunk 1]
```diff
@@ -336,24 +335,24 @@ class EntityHandler(
...
+
+        val updateResult = entityAttributeService.partialUpdateEntityAttribute(entityUri, expandedPayload, contexts)
+
+        if (updateResult.updated.isEmpty())
+            throw ResourceNotFoundException("Unknown attribute in entity $entityId")
+        else
+            entityEventService.publishPartialUpdateEntityAttributesEvents(
+                entityUri,
+                expandedPayload,
+                updateResult,
```
Comment (bobeal) [inline entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt:351]: you could call it directly with the `updateResult.updated` list

File: entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt [hunk 2]
```diff
@@ -1152,20 +1152,84 @@ class EntityHandlerTests {
...
                 {
                     "id": "$entityId",
-                    "type": "DeadFishes"
+                    "type": "DeadFishes",
+                    "@context": ["$AQUAC_COMPOUND_CONTEXT"]
                 }
             """.trimIndent()
         val attrId = "fishNumber"
-
+        val updateResult = UpdateResult(
```
Comment (bobeal) [inline entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt:1212]: I don't get how you can get 2 results whereas you update one attribute instance

Comment (HoucemKacem) [reply entity-service/src/test/kotlin/com/egm/stellio/entity/web/EntityHandlerTests.kt:1212]: Indeed, it should be fixed, the updateResult should only contain one UpdatedDetails element.

File: entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt (hunk 1 above)
Comment (HoucemKacem) [reply entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt]: Done

**Review by bobeal — APPROVED** [review]

**Pull request merged**